build.gradle

```diff
@@ -16,24 +16,24 @@
 import org.gradle.api.internal.plugins.osgi.OsgiHelper
 
 buildscript {
-  ext.kotlinVersion = '1.2.51'
+  ext.kotlinVersion = '1.3.31'
   repositories {
 	maven { url "https://repo.spring.io/plugins-release" }
   }
   dependencies {
 	classpath 'org.springframework.build.gradle:propdeps-plugin:0.0.7'
 	classpath 'io.spring.gradle:spring-io-plugin:0.0.4.RELEASE'
-	classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.0'
+	classpath "com.github.jengelman.gradle.plugins:shadow:4.0.2"
+	classpath "me.champeau.gradle:jmh-gradle-plugin:0.4.8"
 	classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:${kotlinVersion}"
   }
 }
 
 plugins {
   id 'org.asciidoctor.convert' version '1.5.9.2'
-  id "me.champeau.gradle.jmh" version "0.4.7" apply false
   id "me.champeau.gradle.japicmp" version "0.2.6"
   id "de.undercouch.download" version "3.4.3"
-  id "org.unbroken-dome.test-sets" version "1.5.1" apply false // 2.x version does not work with Gradle 4.10
+  id "org.unbroken-dome.test-sets" version "2.1.1" apply false // 2.x version does not work with Gradle 4.10
```

Comment:
despite what is stated compatibility-wise, @bsideup said 2.x was not compatible. the gradlew version is still Gradle 4.10, is it not?

also, consider splitting the commit in 2 (see above)

Reply:
that might be an issue i have a local gradle 5.3 i might not have used 4.10

Reply:
*always* use `./gradlew ...` not `gradle` to avoid that

Reply:
that said, if core _can_ be bumped to 5.3 *in a separate commit* I'm all for it ;)
(provided no plugin and tooling task breaks)